src/app/services/actions/permissions-action-creator.ts

```diff
@@ -285,13 +293,45 @@ export function revokeScopes(permissionToRevoke: string) {
       else {
         const { code, message } = errorMessage;
         trackRevokeConsentEvent(REVOKE_STATUS.failure, 'Failed to revoke consent');
-        dispatchRevokeScopesStatus(dispatch, message ? message : 'Failed to revoke consent', code ? code : 'Failed', 1);
```

Comment:
Does the end user see this message? If so, is there a way to show a clearer error message to them? I find that just telling the user that an action failed doesn't help them much. i.e. why did it fail? What actions can the user take to prevent the failures? Even a suggestion on possible actions could be a great help.

Reply:
The dispatchRevokeScopesStatus method translates this message to this:  "Failed to revoke consent": "An error occurred when unconsenting. Please try again",

What the end user sees is 'An error occurred when unconsenting. Please try again'.